sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/MassageModeData.java

```diff
@@ -0,0 +1,68 @@
+package com.smartdevicelink.proxy.rpc;
+
+import com.smartdevicelink.proxy.RPCStruct;
+import com.smartdevicelink.proxy.rpc.enums.MassageMode;
+import com.smartdevicelink.proxy.rpc.enums.MassageZone;
+
+import java.util.Hashtable;
+
+/**
+ * Specify the mode of a massage zone.
+ */
+
+public class MassageModeData extends RPCStruct{
+    public static final String KEY_MASSAGE_ZONE = "massageZone";
+    public static final String KEY_MASSAGE_MODE = "massageMode";
+
+    /**
+     * Constructs a new MassageModeData object
+     */
+    public MassageModeData() { }
```

Comment:
We started to follow a best practice to add a constructor that has all mandatory params (issue 154). Also, we started adding @nonnull annotation before all mandatory params in that constructor.